must-gather/Dockerfile

```diff
@@ -20,6 +20,7 @@ FROM quay.io/openshift/origin-must-gather:latest AS builder
 
 FROM registry.access.redhat.com/ubi9-minimal:latest
 
+RUN echo -ne "[centos-9-appstream]\nname = CentOS 9 (RPMs) - AppStream\nbaseurl = https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/\nenabled = 1\ngpgcheck = 0" > /etc/yum.repos.d/centos-9-appstream.repo
```

Comment:
How about:
1. Create file `centos-appstream.repo`:

```ini
[centos-9-appstream]
name=CentOS 9 (RPMs) - AppStream
baseurl=https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/
gpgcheck=0
enabled=1
```

2. Inside the Dockerfile instead of `RUN` do:
```
COPY centos-appstream.repo /etc/yum.repos.d/
RUN dnf clean all && dnf makecache
```

Reply:
There is `microdnf` command, may happen it's not compatible with `clean all` or `makecache` but those are good practice and not always required.

Reply:
can you please apply this changes in follow up PR, @mpryc ?

Reply:
@mateusoliveira43 I think you can follow up the PR :)